src/transformers/trainer_tf.py

```diff
@@ -93,11 +92,6 @@ def __init__(
             None,
         ),
     ):
-        assert parse(tf.__version__).release >= (2, 2, 0), (
```

Comment:
Should that be updated to 2.3.0 in this case then?

Reply:
No, because the min version for running the trainer was 2.2.0. So this assert becomes useless.